ci/testsettings.py

```diff
@@ -28,16 +28,15 @@
 
 # required for integration tests that query Solr
 SOLR_CONNECTIONS = {
-    'test': {
-        'COLLECTION': 'test-ppa',
-        'URL': 'http://localhost:8983/solr/',
-        'ADMIN_URL': 'http://localhost:8983/solr/admin/cores'
-    },
     'default': {
-        'COLLECTION': 'ppa-pa11y',
         'URL': 'http://localhost:8983/solr/',
-        'ADMIN_URL': 'http://localhost:8983/solr/admin/cores'
-    },
+        'COLLECTION': 'ppa-pa11y',
```

Comment:
we should probably just call the default core `ppa`; I imagine `pa11y` will get removed before long.

Reply:
I wondered about that; wasn't sure if something depended on it but now that I'm checking I can't find any references. I'll rename.

Reply:
Renaming caused an error, and it didn't seem trivial to update, so I just reverted the change. We can deal with it when we switch from pa11y to axe.